src/nspawn/nspawn-register.h

```diff
@@ -10,6 +10,7 @@
 
 typedef enum RegisterMachineFlags {
         REGISTER_MACHINE_KEEP_UNIT = 1 << 0,
+        REGISTER_MACHINE_ENABLE_FUSE = 1 << 1,
```

Comment:
we usually align flags enums along the `=` these days. i.e please add one extra space before the first enum value here.

Reply:
Done.